package.json

```diff
@@ -34,7 +34,6 @@
     "@hapi/joi": "^16.1.7",
     "bcryptjs": "^2.4.3",
     "body-parser": "^1.19.0",
-    "circular-json": "^0.5.9",
```

Comment:
what is this?, y pq se borra?

Reply:
eso lo agregue cuando estaba trabajando en el signin con git... ya no va. (no recuerdo para que era) :)